.env.example

```
@@ -43,4 +43,7 @@ NPG_TCP_KEEPINTVL="2"
 NPG_TCP_KEEPCNT="10"
 JWT_ISSUER_URI="http://localhost:8098/pagopa-jwt-issuer-service"
 JWT_ISSUER_READ_TIMEOUT=10000
-JWT_ISSUER_CONNECTION_TIMEOUT=10000
\ No newline at end of file
+JWT_ISSUER_CONNECTION_TIMEOUT=10000
+SECURITY_API_KEY_PRIMARY=KEY
+SECURITY_API_KEY_SECONDARY=KEY
+SECURITY_API_KEY_SECURED_PATHS=PATHS
```

```suggestion
SECURITY_API_KEY_PRIMARY=primary-key
SECURITY_API_KEY_SECONDARY=secondary-key
SECURITY_API_KEY_SECURED_PATHS=/payment-methods,/v2/payment-methods
```

we can use "real values" (at leasts paths) also for local tests too, just to make this logic work also locally